packages/template-retail-react-app/app/components/search/index.test.js

```diff
@@ -30,15 +32,6 @@ test('renders SearchInput', () => {
     expect(searchInput).toBeInTheDocument()
 })
 
-test('renders Popover if recent searches populated', async () => {
-    renderWithProviders(<SearchInput />)
-    const searchInput = document.querySelector('input[type="search"]')
-    await user.type(searchInput, 'Dresses')
-    expect(await screen.findByTestId('sf-suggestion')).toBeInTheDocument()
-    const countOfSuggestions = await screen.findAllByText('Dresses')
-    expect(countOfSuggestions.length).toEqual(2)
-})
```

Comment:
I've removed this test as it seems it's sort of a duplicate of "_shows previously searched items when focused_" (L45).